docs/mkdocs.yml

```diff
@@ -1,87 +1,39 @@
-site_name: onosproject
-site_url: https://www.onosproject.org
-site_description: Open Network Operating System (ONOS)
-site_author: Open Networking Foundation
-
 dev_addr: 0.0.0.0:8000
-
+docs_dir: content
+edit_uri: ""
+extra:
+  social:
+  - link: https://github.com/onosproject
+    type: github
+  - link: https://onosproject.slack.com/
+    type: slack
+  - link: https://www.youtube.com/channel/UCTQaOpqno48GTimdwyfytFw
+    type: youtube
+extra_javascript:
+- theme/js/onos-menu.js
```

Comment:
does this need to be in 4 times?

Reply:
I was changing the code and it was added by mistake. I removed them and it should be OK now. Thanks for catching it.